### PR TITLE
Config seeder

### PR DESCRIPTION
**Pull request by cleung2010**
Add the ability to call config_seeder directly from within git2consul if a config_file is passed in.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+v0.12.5
+
+* Configuration file can be directly seeded from within git2consul without calling untils/config_seeder.js
```
Comment (ryanbreen) [inline CHANGELOG.md]: "utils" not "untils"

**Comment by ryanbreen** [comment]
:+1:

**Comment by cleung2010** [comment]
Going to update the docs, might need a final review.

**Comment by cleung2010** [comment]
It all should be good now! Need a second thumbsup for confirmation/sanity checking.

**Comment by ryanbreen** [comment]
:+1:

**Pull request merged**